### PR TITLE
Unit Test Chnages

### PR DESCRIPTION
**Pull request by ndinakar**
@reachrama Hi Rama, Can you please review it, if anything else need to add please let me know. Thanks in advance.

### CODE REVIEW

File: src/main/java/org/recap/camel/dailyreconciliation/DailyReconciliationProcessor.java [hunk 1]
```diff
@@ -162,8 +162,7 @@ public void buildRequestsRows(XSSFWorkbook xssfWorkbook, XSSFSheet xssfSheet, XS
         cellStyle.setAlignment(HorizontalAlignment.LEFT);
         if(StringUtils.isNotBlank(requestId)){
             Optional<RequestItemEntity> requestItemEntity = requestItemDetailsRepository.findById(Integer.valueOf(requestId));
-            if(requestItemEntity != null){
-                ItemEntity itemEntity = requestItemEntity.get().getItemEntity();
+                ItemEntity itemEntity = requestItemEntity.get().getItemEntity();//get().getItemEntity();
```
Comment (reachrama) [inline src/main/java/org/recap/camel/dailyreconciliation/DailyReconciliationProcessor.java]: You have removed null check for optional that's good. But you need to check whether it is present or not. So please use isPresent(), before getting data from optional

File: src/test/java/com/pkrete/jsip2/messages/SIP2ResponseFactoryUT.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        }
+        try {
+            sipResponseFactory.create("18Test");
+        } catch (Exception e) {
+        }
+        try {
+            sipResponseFactory.create("20Test");
+        } catch (Exception e) {
+        }
+        try {
```
Comment (reachrama) [inline src/test/java/com/pkrete/jsip2/messages/SIP2ResponseFactoryUT.java:64]: Is this test method really needed? If not please remove it. And why there is nothing in the exception block

File: src/test/java/com/pkrete/jsip2/parser/SIP2ItemInformationResponseParserUT.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import org.junit.Test;
+import org.recap.BaseTestCase;
+
+import static org.junit.Assert.assertNotNull;
+
+public class SIP2ItemInformationResponseParserUT extends BaseTestCase {
+  /*  @Test
+    public void testParse() {
+        SIP2ItemInformationResponse SIP2ItemInformationResponse = null;
+        SIP2ItemInformationResponseParser sip2ItemInformationResponseParser = new SIP2ItemInformationResponseParser();
```
Comment (reachrama) [inline src/test/java/com/pkrete/jsip2/parser/SIP2ItemInformationResponseParserUT.java:18]: Remove the commented out code or remove the file itself if it is not needed

File: src/test/java/org/recap/activemq/JmxHelperUT.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
+package org.recap.activemq;
```
Comment (reachrama) [inline src/test/java/org/recap/activemq/JmxHelperUT.java:1]: Remove the class, if there is no test methods

File: src/test/java/org/recap/callable/LasItemStatusCheckPollingCallableUT.java [hunk 5]
```diff
@@ -0,0 +1,23 @@
+package org.recap.callable;
+
```
Comment (reachrama) [inline src/test/java/org/recap/callable/LasItemStatusCheckPollingCallableUT.java]: Remove the class, if there is no test methods

File: src/test/java/org/recap/ils/model/nypl/patron/_84UT.java [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+import org.recap.BaseTestCase;
+
+import static org.junit.Assert.assertTrue;
+
+public class _84UT extends BaseTestCase {
+
+    _84 inst;
+
+    @Test
+    public void test_84() {
```
Comment (reachrama) [inline src/test/java/org/recap/ils/model/nypl/patron/_84UT.java:13]: I see the below code chunk is being duplicated a lot, except that instance. Create a base class for _84 like, and have this common changes moved there

**Review by reachrama — COMMENTED** [review]
@DinakarCiber - Please incorporate the comments.

@NSSuresh11 / @rajeshbabuk

**Comment by ndinakar** [comment]
@reachrama  Hi Rama, wherever possible i resolved Issues. Can you please merge it. Thanks in advance.

**Review by reachrama — APPROVED** [review]
okay

**Pull request merged**